### ISSUE TITLE
[Bug]: Windows host not appearing in sections/charts randomly.

### ISSUE THREAD

**Issue by brodonalds** [netdata/netdata#15514]
### Bug description

tl:dr windows node not showing in sections/charts

trying to onboard a windows host but running into an issue. Error is that the machine and any related sections are not appearing any more. They appeared for 1-2 days before disappearing again. I logged into the host and metrics are visible, health check is ok and the exporter is running. From the linux agent side i validated the windows conf file contains the name and url/ip of the machine to reach out to. no errors related to the windows host appear in any of the logs. i checked the files with edit-config. i ran the go plugin in debug go.d.plugin -d -m windows and it shows a wmi.conf being loaded and not the windows.conf.
vnode enabled to try and separate it out as well

### Expected behavior

Click my node and scroll down to find windows and mssql as a section that i can click and view.

### Steps to reproduce

1. login
2. click node
3. scroll to find windows missing
...

### Installation method

kickstart.sh

### System info

```shell
/etc/os-release:NAME="Amazon Linux"

/etc/os-release:VERSION="2"

/etc/os-release:ID="amzn"

/etc/os-release:ID_LIKE="centos rhel fedora"

/etc/os-release:VERSION_ID="2"

/etc/os-release:PRETTY_NAME="Amazon Linux 2"

/etc/os-release:ANSI_COLOR="0;33"

/etc/os-release:CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"

/etc/system-release:Amazon Linux release 2 (Karoo)
```

### Netdata build info

```shell
agent version v1.41.0-40-g604aa6c3c
```

### Additional info

Right now, there is a single linux node that is reporting fine. There is one windows node that was working and then stopped. Vnode creation made no difference. I suspect for some reason, the windows.conf file is not being read.

**Comment by ilyam8** [comment]
Hi, @brodonalds. `wmi.conf` is preferred over `windows.conf` only if it is located in the user config directors (`/etc/netdata/go.d/`). The collector has been renamed from `wmi` to `windows` and prefer the `wmi.conf` file was necessary for backward compatibility.

**Comment by brodonalds** [comment]
So is the fix to move the data to the wmi file? Was this a recent change as this had been working a bit ago.

**Comment by ilyam8** [comment]
It wasn't a recent change. Remove wmi.conf if you don't need it.

**Comment by ilyam8** [comment]
Seems to be [resolved(discrod link)](https://discord.com/channels/847502280503590932/1131295926506491995/1134235233743548538).
